Fakebook.Posts/Fakebook.Posts.UnitTests/CommentTest.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Fakebook.Posts.Domain.Models;
+using Xunit;
+
+namespace Fakebook.Posts.UnitTests
+{
+    public class CommentTest
+    {
+        private const int Id = 1;
+
+        [Fact]
+        public void Comment_Constructor_Pass()
+        {
+            //arrange
+            const string Content = "This is some content";
+            const string userEmail = "damion.silver@test.com";
+            //act
+            Comment newcomment = new Comment(userEmail, Content);
+
+            //assert
+            Assert.NotNull(newcomment);
+        }
+
+        [Fact]
+        public void Comment_EmailIsNotNull()
+        {
+            //arrange
+            const string Content = "This is some content";
+            const string userEmail = "damion.silver@test.com";
+            //act
+            Comment newcomment = new Comment(userEmail, Content);
+
+            //assert
+            Assert.NotNull(newcomment.UserEmail);
+        }
```

Comment:
this is probably overdoing it, this test is pretty redundant given the presence of the next test.

Reply:
I deleted this method.